scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -49,46 +48,12 @@
         <div class="row">
           <div class="col-sm-4">
               <h4 class="">
-                Case: {{case.display_name}} <span class="badge bg-sm bg-secondary" {% if case.lims_id %} title="Case LIMS id {{ case.lims_id }}" data-bs-toggle="tooltip" {% endif %}>{{case._id}}</span><br>
+                 Case: {{case.display_name}} <span class="badge bg-sm bg-secondary">{{case._id}}</span><br>
+              </h4>
+              <h4 class="h4 d-inline-block">
                 Status: {{case.status}}
               </h4>
-              <br>
-              <form method="POST" action="{{ url_for('cases.status', institute_id=institute._id, case_name=case.display_name) }}">
-                  Set:
-                  <div class="btn-group">
-                    <!-- archived/active-->
-                    {% if case.status != 'archived' %}
-                      <button type="button" class="btn btn-warning btn-sm" data-bs-toggle="modal" data-bs-target="#archive_modal">
-                        Archived
-                      </button>
-                    {% else %}
-                      <button type="submit" class="btn btn-warning btn-sm" name="status" value="active">
-                        Unarchived
-                      </button>
-                    {% endif %}
-
-                    <!-- solved/active -->
-                    {% if case.status != 'solved' %}
-                      <button type="button" class="btn btn-success btn-sm" data-bs-toggle="modal" data-bs-target="#solve_modal">
-                        Solved
-                      </button>
-                    {% else %}
-                      <button type="submit" class="btn btn-success btn-sm" name="status" value="active">
-                        Unsolved
-                      </button>
-                    {% endif %}
-
-                    <!-- prioritized/active -->
-                    <button name="status" value="{{ 'active' if case.status == 'prioritized' else 'prioritized' }}" type="submit" class="btn btn-info btn-sm">
-                      {{ 'De-prioritized' if case.status == 'prioritized' else 'Prioritized' }}
-                    </button>
-
-                      <!-- unused/inactive -->
-                    <button name="status" value="{{ 'inactive' if case.status == 'ignored' else 'ignored' }}" type="submit" class="btn btn-light btn-sm">
-                      {{ 'Inactive' if case.status == 'ignored' else 'Ignored' }}
-                    </button>
-                  </div>
-              </form>
+              {% for status_tag in case.tags %}<span class="badge bg-info align-top" data-bs-toggle="tooltip" data-bs-placement="right" title="{{ case_tag_options[status_tag].description }}">{{ case_tag_options[status_tag].label }}</span>{% endfor %}
```

Comment:
Perhaps it would be better if the spans are aligned in the middle, as also suggested by Terese?

<img width="467" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/0eac3d20-b12a-4f80-b421-689dead3da42">